services.yaml

```diff
@@ -4784,6 +4830,60 @@ definitions:
         items:
           type: string
           example: MyApp
+  Agent:
+    description: Agent properties
+    type: object
+    properties:
+      server:
+        type: string
+        description: name of the server that hosts the agent
+        example: ip-10-164-178-61
+      app:
```

Comment:
should we say app? not application?

Reply:
yep, application will be right here (that's what we use across the spec)